apps/coscrad-frontend/src/app/App.tsx

```diff
@@ -39,6 +41,14 @@ export function App() {
                         path={routes.resources.ofType(ResourceType.term).detail()}
                         element={<TermDetailContainer />}
                     />
+                    <Route
```

Comment:
We may want to have a helper that bootstraps the routes for resources. this is kind of repetitive and tough to read.